requirements.txt

```diff
@@ -15,7 +15,7 @@ orjson~=3.0
 packaging
 pandas>=1.0,<3.0
 pillow>=8.0,<12.0
-pydantic>=2.0,<=2.12.3
+pydantic>=2.11.10,<=2.12.4
```

Comment:
2.11.9 and earlier versions of `pydantic` have a bug with `mcp`. Hopefully this is not too restrictive.